### PR TITLE
Fix for #408

### PR DESCRIPTION
**Pull request by pralhadsapre**

### CODE REVIEW

**Comment by pralhadsapre** [comment]
I fixed the compilation issues found during install testing. The problem was related to setting the C++ standard in CMake.

**Review by juanlu-rti — APPROVED** [review]

File: examples/routing_service/routing_service_struct_array_transf/cpp/CMakeLists.txt [hunk 1]
```diff
@@ -130,4 +145,14 @@ set_target_properties(SensorDataSubscriber
...
+        LIBRARY_OUTPUT_DIRECTORY "${output_dir}"
+        LIBRARY_OUTPUT_DIRECTORY_RELEASE "${output_dir}"
+        LIBRARY_OUTPUT_DIRECTORY_DEBUG "${output_dir}"
+        ARCHIVE_OUTPUT_DIRECTORY "${output_dir}"
+        ARCHIVE_OUTPUT_DIRECTORY_RELEASE "${output_dir}"
+        ARCHIVE_OUTPUT_DIRECTORY_DEBUG "${output_dir}"
+        RUNTIME_OUTPUT_DIRECTORY "${output_dir}"
+        RUNTIME_OUTPUT_DIRECTORY_RELEASE "${output_dir}"
+        RUNTIME_OUTPUT_DIRECTORY_DEBUG "${output_dir}"
+)
```
Comment (lulivi) [inline examples/routing_service/routing_service_struct_array_transf/cpp/CMakeLists.txt:158]:
```suggestion
)

```
Please, add an endline!

**Review by lulivi — CHANGES_REQUESTED** [review]
Just that little change!

File: examples/routing_service/routing_service_struct_array_transf/cpp/CMakeLists.txt (hunk 1 above)
Comment (pralhadsapre) [reply examples/routing_service/routing_service_struct_array_transf/cpp/CMakeLists.txt:158]: Done.

**Comment by pralhadsapre** [comment]
@lulivi I think there is some issue with the linter. I am seeing this message - ``Configuration file(s) do(es) not support C++: /home/runner/work/rticonnextdds-examples/rticonnextdds-examples/.clang-format``.

**Comment by lulivi** [comment]
We are fixing it right now!

**Review by lulivi — APPROVED** [review]

**Comment by lulivi** [comment]
@pralhadsapre can you merge develop into your branch to see if the linter runs with the new config?

**Comment by lulivi** [comment]
Tested locally and works as expected:

image

File: examples/routing_service/routing_service_struct_array_transf/cpp/CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# (c) 2018 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+# (c) 2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
```
Comment (angelrti) [inline examples/routing_service/routing_service_struct_array_transf/cpp/CMakeLists.txt:2]: It's not needed to update the copyright year, so, don't worry too much about these changes

**Review by angelrti — APPROVED** [review]

**Pull request merged**